src/activity_item_handling.cpp

```diff
@@ -1084,7 +1084,7 @@ static std::pair<bool, do_activity_reason> can_do_activity_there( const activity
                 b_rack_present = true;
             }
         }
-        if( corpses.size() > 0 ) {
+        if( corpses.empty() ) {
```

Comment:
This inverts the meaning of the condition. And it makes the checks for `big_count > 0` below always false, so it does not seem to be a correct change.